static/js/admin.js

```diff
@@ -131,14 +133,38 @@ function populate_users(realm_people_data) {
 
     var bots_table_html = "";
     _.each(bots, function (user) {
-        var bot_html = templates.render("admin_user_list", {user: user});
+        var bot_html = templates.render("admin_user_list", {
+            user: {
+                full_name: user.full_name,
+                email: user.email,
+                bot_owner: user.bot_owner,
+                is_active: user.is_active,
+                is_bot: user.is_bot,
+                is_admin: user.is_admin,
+                is_active_human: user.is_active_human,
+                user_id: user.user_id,
+                is_user_admin: page_params.is_admin,
+            },
```

Comment:
Instead of changing it like this, I would do it something like this:
```var bot_html = templates.render("admin_user_list", {user: user, is_user_admin: page_params.is_admin});```
and then use `../` segment in handlebars template to refer to it.(See handlebars paths on handlebarsjs.com)